packages/react-composites/src/composites/common/styles/Pane.styles.ts

```diff
@@ -75,6 +75,7 @@ export const hiddenStyles: IStackStyles = {
 export const sidePaneStyles: IStackStyles = {
   root: {
     height: 'auto',
+    width: '100%',
```

Comment:
Can we test the scenairo that when the width and height is not set for parent container - just simply remove width = 100% height=100% of .root class in callWithChat sample app and test it

Reply:
When root width and height are unset, the height does shrink:
<img src="https://user-images.githubusercontent.com/97130533/208157673-6c0807d8-c10c-4a88-8162-09c46aba3298.png" width="50%"/>